utils/throttler_test.go

```diff
@@ -13,18 +15,22 @@ import (
 func TestThrottler(t *testing.T) {
```

Comment:
In practice this test was working fine, but (and I agree with -race) it could lead to race conditions